gnomad/utils/release.py

```diff
@@ -92,3 +94,34 @@ def _get_index(label_groups):
         )
 
     return index_dict
+
+
+def make_freq_index_dict_from_meta(
+    freq_meta: List[Dict[str, str]],
+    label_delimiter: str = "_",
+    sort_order: Optional[List[str]] = SORT_ORDER,
+) -> Dict[str, int]:
+    """
+    Create a dictionary for accessing frequency array.
+
+    :param freq_meta: List of dictionaries containing frequency metadata.
+    :param label_delimiter: Delimiter to use when joining frequency metadata labels.
+    :param sort_order: List of frequency metadata labels to use when sorting the dictionary.
+    :return: Dictionary of frequency metadata.
+    """
+    index_dict = {}
```

Comment:
Maybe add a warning if there are labels in freq_meta that are not in sort_order and if there are labels in sort_order that are not in freq_meta